package.json

```diff
@@ -33,7 +33,11 @@
     "uglify-js": "2.7.0"
   },
   "dependencies": {
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-env": "^1.7.0",
```

Comment:
I don't want to introduce babel to this project. I'd like the option of merging this back into the main repo so I don't want to add dependencies unless I need them.